x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -61,4 +62,19 @@ export function initServicesApi(core: CoreSetup) {
       return getService(serviceName, setup).catch(defaultErrorHandler);
     }
   });
+
+  server.route({
```

Comment:
Would it make sense to make a route called `/filters`? Eg `api/apm/filters/environments` and `api/apm/filters/hosts` etc.

I think that co-location would be nice.